README.md

```diff
@@ -2,10 +2,22 @@
 
 Repositório do site da Lyseon Tech
 
+Primeiro clone o repositório: git clone https://github.com/LyseonTech/site.git
+
 ## Build
 
-    composer install
-    ./vendor/bin/sculpin generate --watch --server
+	- Crie uma pasta para o Jigsaw
+		- Linux Users: mkdir my-site && cd my-site
+
+	- composer require tightenco/jigsaw	
+
+	- jigsaw init
+
+	-Dentro da pasta do site da Lyseon Tech rode o comando: composer install
+
+	-Após rodar o comando anterior rode: jigsaw build
```

Comment:
O comando `jigsaw build` só funcionará se a instalação do Jigsaw for feita globalmente e o arquivo executável do jigsaw estiver no bash da sessão do terminal aberta. Da forma como o Jigsaw está em nosso projeto, após executar `composer install` o Jigsaw estará em `vendor/bin/jigsaw` e a forma mais indicada de executá-lo para _buildar_ o projetoseria estar na raiz do projeto e executar `./vendor/bin/jigsaw build` fazendo referência a instalação do Jigsaw no próprio projeto.

Por conta de trabalharmos com integração contínua, o mais aconselhável é utilizar o executável do Jigsaw dentro do próprio projeto.